src/core/control/LatexController.cpp

```diff
@@ -310,7 +311,7 @@ auto LatexController::loadRendered(string renderedTex) -> std::unique_ptr<TexIma
     img->setX(posx);
     img->setY(posy);
     img->setText(std::move(renderedTex));
-    if (imgheight) {
+    if (std::abs(imgheight) > std::numeric_limits<double>::epsilon()) {
```

Comment:
Generally very good change, honestly we need some helper structs for floating point operators. 
But I would increase the epsilon at minimum by a factor of 1000 or 1024 since epsilon corresponds to 1.0 (is the difference between 1.0 and the next representable successor).

And I think the range could be between 0 and 10000(?), the maximum error can also increase for each mathematical operation done before in the worst case: E_worst= for all operations Sigma E(operation)